{{cookiecutter.project_slug}}/requirements/base.txt

```diff
@@ -24,7 +24,7 @@ flower==0.9.5  # https://github.com/mher/flower
 {%- endif %}
 {%- endif %}
 {%- if cookiecutter.use_async == 'y' %}
-uvicorn==0.12.2  # https://github.com/encode/uvicorn
+uvicorn[standard]==0.12.2  # https://github.com/encode/uvicorn
 wsproto==0.15.0  # https://github.com/python-hyper/wsproto/
```

Comment:
Do we still need `wsproto`? I think it's just [a fallback to `websocket`](https://github.com/encode/uvicorn/blob/a839da2564e5168c6e99c3b22819950f26f7d178/uvicorn/protocols/websockets/auto.py#L1-L5) which gets installed with the `standard` extra. So it's now effectively unused...

```suggestion
uvicorn[standard]==0.12.2  # https://github.com/encode/uvicorn
```

Reply:
tl;dr It might just be best to remove wsproto.

If you want to block initial connection without accepting (e.g. because user is not authenticated), you'll have no choice but to use websockets. If you accept then immediately close the connection (making that two messages sent from the server rather than one), then you can use wsproto until they fix the issue I raised in their repo.